composer.json

```diff
@@ -16,18 +16,24 @@
         "forum": "https://discourse.laminas.dev"
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "platform": {
+            "php": "8.0.99"
+        },
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        }
     },
     "require": {
-        "php": "~7.4.0 || ~8.0.0 || ~8.1.0",
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
         "ext-dom": "*",
         "ext-soap": "*",
         "laminas/laminas-server": "^2.11",
         "laminas/laminas-stdlib": "^3.6",
         "laminas/laminas-uri": "^2.9.1"
     },
     "require-dev": {
-        "laminas/laminas-coding-standard": "~2.2.1",
+        "laminas/laminas-coding-standard": "~2.4",
```

Comment:
Overall OK to bump the CS ruleset, but we need to exclude the strict standards rules, or else we run into massive BC breaks.

Can you run `./vendor/bin/phpcs -s` and then modify `phpcs.xml.dist` so that the strict standards rule (you'll see its name in the output) is suppressed?

Reply:
Multiple files are there after running ./vendor/bin/phpcs -s

FILE: src/Exception/UnexpectedValueException.php
----------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------------------------------------------------------------------
 1 | ERROR | [x] Missing declare(strict_types=1).
   |       |     (SlevomatCodingStandard.TypeHints.DeclareStrictTypes.DeclareStrictTypesMissing)
----------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------------------------------------------------------------------


FILE: src/Wsdl/ComplexTypeStrategy/AnyType.php
----------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------------------------------------------------------------------
 1 | ERROR | [x] Missing declare(strict_types=1).
   |       |     (SlevomatCodingStandard.TypeHints.DeclareStrictTypes.DeclareStrictTypesMissing)
----------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------------------------------------------------------------------


FILE: src/Exception/ExtensionNotLoadedException.php
----------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------------------------------------------------------------------
 1 | ERROR | [x] Missing declare(strict_types=1).
   |       |     (SlevomatCodingStandard.TypeHints.DeclareStrictTypes.DeclareStrictTypesMissing)
----------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------------------------------------------------------------------


FILE: src/Exception/ExceptionInterface.php
--------------------------------------------


**ETC.** 

What need to do in it?

Reply:
Handled in 67e4e34